docker-compose.yml

```
@@ -0,0 +1,19 @@
+version: '3.7'
+
+services:
+  frontend:
+    restart: always
+    build:
+      context: ./
+      dockerfile: ./Dockerfile
+      target: base
+    env_file: .env
+    volumes:
+      - ./:/code
+      # https://github.com/BretFisher/node-docker-good-defaults/blob/b13c648411335fbd6927b4421319e1bf2dad05a3/docker-compose.yml#L29
+      # prevent host node_modules from accidently getting mounted in container
+      - /code/node_modules
+    ports:
+      - '3000:3000'
+    stdin_open: true
+    command: ash -c "yarn start"
```

With `ash -c` it throws
```
frontend_1  | $ webpack-dev-server --config webpack.dev.js --host 0.0.0.0
frontend_1  | /bin/sh: webpack-dev-server: not found
frontend_1  | error Command failed with exit code 127.
```
to me. Without it works nice.
```suggestion
    command: "yarn start"
```